gammacat/webpage.py

```diff
@@ -20,11 +23,60 @@ def __init__(self, *, out_path):
 
 
 class WebpageMaker:
-    """Make gamma-cat webpage."""
+    """Make rst-files for gamma-cat webpage."""
 
     def __init__(self, config):
         self.config = config
+        self.sources_data = BasicSourceList.read().to_dict()['data']
 
     def run(self):
         log.info('Make webpage ...')
-        log.error('Come one. Implement me already, you lazy hog!!!')
+
+        self.make_source_list()
+        self.make_sources()
+
+    def make_source_list(self):
+
+        status_rst = str(gammacat_info.base_dir / 'documentation/data/src_list.rst')
+        #TODO: Shift the template things to a function in utils.py
+        templateLoader = jinja2.FileSystemLoader( searchpath = '/')
```

Comment:
On Slack I gave you the render helper function on Friday. Please add it to utils and then call it from webpage.py